authprogs/tests/test_authprogs.py

```diff
@@ -133,6 +134,12 @@ def test_client_ip(self):
         ap = getap('0.0.0.1', 'MULTIPLE_TWO')
         self.assertRaises(authprogs.CommandRejected, ap.find_match)
 
+    def test_client_ip_cidr(self):
+        """Test that a single bare command works."""
+        ap = getap('10.11.12.13', 'CIDR')
+        self.assertEqual(ap.find_match(),
+                {'command': ['CIDR']})
+
```

Comment:
This should also include a failure check, like:

     ap = getap('0.0.0.1', 'CIDR')
     self.assertRaises(authprogs.CommandRejected, ap.find_match)

Reply:
Fair